pkg/kubectl/cmd/create_clusterrole.go

```diff
@@ -133,6 +134,12 @@ func (c *CreateClusterRoleOptions) Validate() error {
 				return fmt.Errorf("invalid verb: '%s' for nonResourceURL", v)
 			}
 		}
+
+		for _, nonResourceURL := range c.NonResourceURLs {
+			if !strings.HasPrefix(nonResourceURL, "/") {
```

Comment:
no server side validation, just make sure there's no more than one `*` here, and if there is one, it is at the end